net-misc/youtube-viewer/youtube-viewer-3.5.8-r1.ebuild

```diff
@@ -35,7 +35,7 @@ RDEPEND="
 		virtual/freedesktop-icon-theme
 		x11-libs/gdk-pixbuf:2[X,jpeg]
 	)
-	|| ( media-video/ffmpeg[openssl] media-video/ffmpeg[gnutls] )
+	|| ( media-video/ffmpeg[openssl,-libressl] media-video/ffmpeg[-openssl,libressl] media-video/ffmpeg[gnutls] )
```

Comment:
About the previous message, I still think that method is better, just update to reflect on ffmpeg. Buuut lets try it with this one for now.
Thanks!

Reply:
I think due to YouTube enforcing SSL, there is no point in having an "ssl" use flag on youtube-viewer, since we don't want to turn off SSL anyway. The only choice the user has is to pick SSL implementation, so we enforce that ffmpeg is built with at least one of them. This is why I chose to implement it this way in this ebuild.